rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -53,6 +51,9 @@ configuration over time.
 
 %install
 %{python3} -m pip install --no-compile --no-index --no-build-isolation --root %{buildroot} .
+# direct_url.json is is not reproducible and not strictly needed
+rm %{buildroot}/%{python3_sitelib}/*%{version}.dist-info/direct_url.json
+sed -i "/\.dist-info\/direct_url\.json,/d" %{buildroot}/%{python3_sitelib}/*%{version}.dist-info/RECORD
```

Comment:
from the client:
```
override_dh_strip_nondeterminism:
	find ./debian/ -type f -name '*.pyc' -delete
	find ./debian/ -type d -name '__pycache__' -delete
	find ./debian/ -type f -name 'pip-selfcheck.json' -delete
	find ./debian/ -type f -name 'direct_url.json' -delete
	find ./debian/ -type f -name 'RECORD' -delete
	dh_strip_nondeterminism $@
```

let's just delete these instead of trying to fix them up